en/docs/install-and-setup/setup/api-controller/managing-microgateways/managing-microgateways-with-ctl.md

```diff
@@ -0,0 +1,293 @@
+# Managing Microgateways
+
+WSO2 API Controller, **apictl** can be used as a developer CLI tool for Microgateway. It enables interacting with one or more Microgateways without having to perform the following tasks via the WSO2 API Manager.
```

Comment:
```suggestion
WSO2 API Controller (apictl) can be used as a developer CLI tool for Microgateway. It enables interacting with one or more Microgateways without having to perform the following tasks via the WSO2 API Manager (WSO2 API-M).
```

Reply:
Shall we change like above? (Especially we are planning to correct all the apictl abbreviation occurences as shown above in the doc space soon)

Reply:
Thanks, will do. Will update other occurrences accordingly.